tools/encodings/GF Glyph Sets/Cyrillic/GF-cyrillic-pro_unique-glyphs.nam

```diff
@@ -1,4 +1,5 @@
 # Created by Alexei Vanyashin 2016-27-06
+#$ include GF-cyrillic-plus_unique-glyphs.nam
 0x048A  Ҋ uni048A
```

Comment:
yeah, I'll do so later. For now it's not so important, because I really only read the unicode codepoint. But I thought about doing so before :-)